liblouis/lou_backTranslateString.c

```diff
@@ -266,6 +266,12 @@ _lou_backTranslate(const char *tableList, const char *displayTableList,
 					appliedRules, &appliedRulesCount, maxAppliedRules);
 			break;
 		}
+		
+		currentPass--;
+		// If not goodTrans, we need to break out of the loop directly, 
+		// as realInlen might be an uninitialized value in this case.
+		if (!goodTrans) break;
```

Comment:
Wouldn't it make more sense to immediately `return 0` in this case?

Reply:
I just looked for the smallest possible semantic change. If it's okay to immediately `return 0`, I'll change it.

Reply:
@benehalo We're looking at it now and think it's safe to just return `0` immediately. It will make the code much simpler to follow.

Reply:
See https://github.com/liblouis/liblouis/pull/1590/commits/2dabab7d7a85de6855a08a43a5a987984538bc9c

Reply:
I dropped the whole code with `goodtrans`. Instead we fail directly if any of the calls above fail.